src/app/models/propTypes/cpsAssetPage/index.js

```diff
@@ -0,0 +1,26 @@
+import { bool, shape, string, number, any } from 'prop-types';
+// import { cpsMetadataPropTypes } from '../metadata';
```

Comment:
can we delete the commented out code, or add a TODO comment if something needs done with it in the future